tools/packaging/common/sidecar.env

```diff
@@ -85,7 +85,7 @@
 
 # Location of provisioning certificates. VM provisioning tools must generate a certificate with
 # the expected SAN. Istio-agent will use it to connect to istiod and get fresh certificates.
-#PROV_CERT=/var/run/secrets/istio
```

Comment:
why not us /var/run? my understanding is this is industry standard to store secrets

Reply:
Now with `#PROV_CERT=/var/run/secrets/istio `, it means user must put related certs 
under '/var/run/secrets/istio` and the auto cert rotation will not happen